rfcs/0048-on-conflict-clause-in-create-table-statement.md

```diff
@@ -0,0 +1,121 @@
+---
+feature: ON CONFLICT Clause in Create Table Statement
+authors:
+  - "st1page"
+start_date: "2023/2/1"
+---
+
+# ON CONFLICT Clause in Create Table Statement
+
+## Summary
+
+allow user to declare the conflict behavior when the newly inserted row break the unique constraint of primary key.
```

Comment:
by the way, when do we do constraint checks? every time a row is materialized?

Reply:
Yes, the logic will be implemented in the materialize executor of the table.